lib/active_merchant/billing/gateways/payu_latam.rb

```diff
@@ -78,6 +78,7 @@ def refund(amount, authorization, options={})
       def verify(credit_card, options={})
         minimum = MINIMUMS[options[:currency].upcase] if options[:currency]
         amount = minimum || 100
+        amount = options[:verify_amount] if options[:verify_amount]
```

Comment:
(Not holding for this.) This can all be combined into `amount = options[:verify_amount] || minimum || 100`.

Reply:
👍 updated